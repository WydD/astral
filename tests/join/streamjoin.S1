id,temp,T
1,30,1000
1,35,1005
1,25,1005
1,29,1005
1,33,1010
1,25,1015
1,25,1015
1,32,1015
