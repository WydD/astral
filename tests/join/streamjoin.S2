id,T,temp2
1,1000,35
1,1005,28
2,1008,28
1,1010,33
1,1015,28
1,1020,30
1,1025,35
1,1030,26
1,1035,34
1,1040,32
1,1045,32
