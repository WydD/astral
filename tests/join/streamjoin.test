<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<streamjoin band="10">
<source attributes="id,T,PHYID,temp" id="S1" sourceAttributes="id,T,PHYID,temp" type="Stream"/>
<source attributes="id,T,temp2,PHYID" id="S2" sourceAttributes="id,T,temp2,PHYID" type="Stream"/>
</streamjoin>
</query>
<sources>
<source attributes="id,T,PHYID,temp" file="tests/join/streamjoin.S1" id="S1" impl="FileStreamImpl" type="Stream"/>
<source attributes="id,T,temp2,PHYID" file="tests/join/streamjoin.S2" id="S2" impl="FileStreamImpl" type="Stream"/>
</sources>
</astral>
