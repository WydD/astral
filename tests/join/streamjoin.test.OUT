id,temp,temp2,T
1,30,35,1000
1,30,28,1005
1,35,35,1005
1,35,28,1005
1,25,35,1005
1,25,28,1005
1,29,35,1005
1,29,28,1005
1,30,33,1010
1,35,33,1010
1,25,33,1010
1,29,33,1010
1,33,35,1010
1,33,28,1010
1,33,33,1010
1,35,28,1015
1,25,28,1015
1,29,28,1015
1,33,28,1015
1,25,28,1015
1,25,33,1015
1,25,28,1015
1,25,28,1015
1,25,33,1015
1,25,28,1015
1,32,28,1015
1,32,33,1015
1,32,28,1015
1,33,30,1020
1,25,30,1020
1,25,30,1020
1,32,30,1020
1,25,35,1025
1,25,35,1025
1,32,35,1025